contracts/ERC721MLite.sol

```diff
@@ -0,0 +1,568 @@
+//SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+import "@openzeppelin/contracts/utils/cryptography/SignatureChecker.sol";
+import "erc721a/contracts/extensions/ERC721AQueryable.sol";
+import "./IERC721M.sol";
+
+/**
+ * @title ERC721MLite
+ *
+ * @dev ERC721A subclass with MagicEden launchpad features including
```

Comment:
nit: add `cosigner` to the list of features in this comment?

Reply:
Good catch.

Reply:
Added it as `anti-botting`